{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "important"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python36-setuptools",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for python36-setuptools fixes the following issues:\n\n- CVE-2024-6345: Fixed code execution via download functions in the package_index module (bsc#1228105)\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "SUSE-2024-2950,SUSE-SLE-SERVER-12-SP5-2024-2950",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2024_2950-1.json"
			},
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2024:2950-1",
				"url": "https://www.suse.com/support/update/announcement/2024/suse-su-20242950-1/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2024:2950-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2024-August/019217.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1228105",
				"url": "https://bugzilla.suse.com/1228105"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2024-6345 page",
				"url": "https://www.suse.com/security/cve/CVE-2024-6345/"
			}
		],
		"title": "Security update for python36-setuptools",
		"tracking": {
			"current_release_date": "2024-08-16T14:03:46Z",
			"generator": {
				"date": "2024-08-16T14:03:46Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2024:2950-1",
			"initial_release_date": "2024-08-16T14:03:46Z",
			"revision_history": [
				{
					"date": "2024-08-16T14:03:46Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python36-setuptools-44.1.1-8.9.1.noarch",
								"product": {
									"name": "python36-setuptools-44.1.1-8.9.1.noarch",
									"product_id": "python36-setuptools-44.1.1-8.9.1.noarch"
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE Linux Enterprise Server 12 SP5",
								"product": {
									"name": "SUSE Linux Enterprise Server 12 SP5",
									"product_id": "SUSE Linux Enterprise Server 12 SP5",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:sles:12:sp5"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE Linux Enterprise Server for SAP Applications 12 SP5",
								"product": {
									"name": "SUSE Linux Enterprise Server for SAP Applications 12 SP5",
									"product_id": "SUSE Linux Enterprise Server for SAP Applications 12 SP5",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:sles_sap:12:sp5"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python36-setuptools-44.1.1-8.9.1.noarch as component of SUSE Linux Enterprise Server 12 SP5",
					"product_id": "SUSE Linux Enterprise Server 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch"
				},
				"product_reference": "python36-setuptools-44.1.1-8.9.1.noarch",
				"relates_to_product_reference": "SUSE Linux Enterprise Server 12 SP5"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python36-setuptools-44.1.1-8.9.1.noarch as component of SUSE Linux Enterprise Server for SAP Applications 12 SP5",
					"product_id": "SUSE Linux Enterprise Server for SAP Applications 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch"
				},
				"product_reference": "python36-setuptools-44.1.1-8.9.1.noarch",
				"relates_to_product_reference": "SUSE Linux Enterprise Server for SAP Applications 12 SP5"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2024-6345",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2024-6345"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "A vulnerability in the package_index module of pypa/setuptools versions up to 69.1.1 allows for remote code execution via its download functions. These functions, which are used to download packages from URLs provided by users or retrieved from package index servers, are susceptible to code injection. If these functions are exposed to user-controlled inputs, such as package URLs, they can execute arbitrary commands on the system. The issue is fixed in version 70.0.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"recommended": [
					"SUSE Linux Enterprise Server 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch",
					"SUSE Linux Enterprise Server for SAP Applications 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2024-6345",
					"url": "https://www.suse.com/security/cve/CVE-2024-6345"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1228105 for CVE-2024-6345",
					"url": "https://bugzilla.suse.com/1228105"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE Linux Enterprise Server 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch",
						"SUSE Linux Enterprise Server for SAP Applications 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 7,
						"baseSeverity": "HIGH",
						"vectorString": "CVSS:3.1/AV:L/AC:H/PR:N/UI:R/S:U/C:H/I:H/A:H",
						"version": "3.1"
					},
					"products": [
						"SUSE Linux Enterprise Server 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch",
						"SUSE Linux Enterprise Server for SAP Applications 12 SP5:python36-setuptools-44.1.1-8.9.1.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2024-08-16T14:03:46Z",
					"details": "important"
				}
			],
			"title": "CVE-2024-6345"
		}
	]
}
